.github/workflows/docs-preview.yml

```diff
@@ -61,9 +61,7 @@ jobs:
         with:
           hugo-version: '0.79.1'
       - name: Build Hugo docs
-        run: |
-          make build
-          HUGO_ENV=production hugo -b "https://docs-preview.oso.dev/"
+        run: make build
```

Comment:
Doesn't this already run on every PR?

Reply:
The YAML was off. Thanks for making me look twice at it.

Reply:
Ohhhh